openslides/users/static/templates/users/group-detail.html

```diff
@@ -0,0 +1,14 @@
+<h1>{{ group.name }}</h1>
+
+<div id="submenu">
+  <a ui-sref="users.group.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
+  </a>
+</div>
+
+
+<b translate>Permissions:</b>
+<ul ng-repeat="perm in group.permissions">
+  <li>{{ perm }}
```

Comment:
An li element's end tag may be omitted if the li element is immediately followed by another li element or if there is no more content in the parent element.

ok